.editorconfig

```diff
@@ -0,0 +1,19 @@
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.php]
+indent_style = space
```

Comment:
Go spaces!  
Good to have an editorconfig file finally, was about time :)

Reply:
Yeah man, I don't think I can work without one now!